docs/source/markdown/podman-tag.1.md

```diff
@@ -19,14 +19,19 @@ the *image* and the *target-name*.
 
 Print usage statement
 
+**--undo**, **-u**
+
+Undo the latest tag operation and restore the previous tag
+
 ## EXAMPLES
 
 ```
 $ podman tag 0e3bbc2 fedora:latest
 
 $ podman tag httpd myregistryhost:5000/fedora/httpd:v2
-```
 
+$ podman tag fedora:latest --undo
```

Comment:
I'd use `0e3bbc2` to illustrate that the ID is static but the tags are somehow arbitrary.

Reply:
Changed as suggested. :+1: